core/src/main/java/com/rudderstack/android/sdk/core/RudderDeviceModeManager.java

```diff
@@ -44,10 +43,10 @@ public class RudderDeviceModeManager {
     private RudderEventFilteringPlugin rudderEventFilteringPlugin;
     private final Map<String, RudderIntegration<?>> integrationOperationsMap;
     private final Map<String, RudderClient.Callback> integrationCallbacks;
-    private final Map<Integer, RudderMessage> eventReplayMessageQueue;
     private final RudderDataResidencyManager dataResidencyManager;
     // required for device mode transform
     private final Map<String, String> destinationsWithTransformationsEnabled = new HashMap<>(); //destination display name to destinationId
+    private boolean isDeviceModeFactoriesNotPresent = false;
```

Comment:
I would prefer to rename this to `areDeviceModesAbsent`  / `areDeviceModeFactoriesAbsent`

Reply:
I've renamed it to `areDeviceModeFactoriesAbsent`.